src/app/layout/navbar/navbar.component.html

```diff
@@ -1,5 +1,12 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <div class="container">
-    <a class="navbar-brand" href="#">GP4Tech</a>
+<nav class="navbar-fixed">
+  <div class="nav-wrapper">
+    <div class="container">
+      <a href="#" class="brand-logo left">GP4Tech</a>
```

Comment:
After this line we can add the hamburguer menu button:
```
<a data-target="mobile-sidenav" class="sidenav-trigger"><i class="material-icons">menu</i></a>
```
The icon doesn't need to be from material icons.

Reply:
@mauricioarce yeap, also this will be added in a second PR with the full functionality of the sidenav